source/_changelogs/4.3.0.md

```diff
@@ -0,0 +1,48 @@
+- When you open a Cypress project, clicking on the **Settings** tab and clicking into the **Experiments** panel will display the {% url "experimental features" configuration#Experiments %} that are available and whether they are enabled for your project.
+- Added support for setting cookie `sameSite` values via `cy.setCookie()`. Addresses {% issue 2437 %}.
+- Added experimental support for adding `sameSite` values to the objects yielded from {% url "`cy.setCookie()`" setcookie %}, {% url "`cy.getCookie()`" getcookie %}, and {% url "`cy.getCookies()`" getcookies %}. Users can enable this by setting the {% url "`experimentalGetCookiesSameSite`" configuration#Experiments %} configuration value to `true`. In Cypress 5, this will be the default. Addresses {% issue 2437 %}.
+- Added support for the `SameSite` cookie attribute in `cy.request()`, `cy.visit()`, and proxied HTTP requests. Addresses {% issue 6757 %}.
+
+**Bugfixes:**
+
+- Fixed a regression in {% url "4.2.0" changelog-4-2-0 %} where cypress installation could fail if Git is not installed. Fixes {% issue 6752 %}.
+- Fixed a regression in {% url "3.5.0" changelog-3-5-0 %} where certain HTTP requests could fail with `Parse Error: Invalid header value char` or `Parse Error: Header overflow`. Fixes {% issue 5602 %}.
+- Fixed a regression in {% url "4.2.0" changelog-4-2-0 %} where Electron deprecation warnings were printed to stderr in open mode. Fixes {% issue 6750 %}.
+- Cookies set with a Domain containing an unknown TLD are now sent along for all requests. For example, a cookie like `foo=bar; Domain=.cypress.test` will now be sent during {% url "cy.request()" request %} to `http://local.cypress.test`. Fixes {% issue 6628 %}.
+- We fixed a situation where a cross-origin errors could incorrectly throw in Chrome. Fixes {% issue 5270 %}.
+- Visibility checks against `vue-fragment` elements no longer throw an unexpected error. Fixes {% issue 6745 %}.
+- The stack trace is now properly shown in the console when clicking the command for tests having an `.only` that throw an error. Fixes {% issue 6744 %}.
+- `firefoxGcInterval` can now correctly be set to `null`. Fixes {% issue 6825 %}.
+- Cypress will no longer crash the process if an empty video chunk is received. Addressed in {% PR 6818 %}.
+
+**Misc:**
+
+- Security warnings no longer show in Mac OS when opening Cypress since our application now undergoes notarization from Apple. Addresses {% issue 5791 %}.
+- When an assertion involving a DOM element retry fails, it now shows a more sensible DOM object in the diff. Addresses {% issue 6723 %}.
+- When an assertion is retried (using {% url "`.should()`" should %}) and fails, it now shows the diff in the `stdout`. Addresses {% issue 6722 %}.
```

Comment:
Move this to `Features:`

Reply:
69aa919a752d6e9bc82b33b10e8ace1c8d153b17